File: src/main/java/com/bot/model/dto/StatisticDto.java
package com.bot.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.io.File;
import java.math.BigDecimal;
import java.util.List;

@Builder
@Data
@AllArgsConstructor
public class StatisticDto {

    private String message;
    private BigDecimal totalSpend;
    private File statisticFile;
    private List<StatisticDataDto> categotySpendings;
}

File: telegram-adapter/src/main/kotlin/com/planner/model/dto/StatisticDto.kt
package com.planner.model.dto

import java.io.File
import java.math.BigDecimal


data class StatisticDto(
    val message: String,
    val totalSpend: BigDecimal,
    val statisticFile: File,
    val categotySpendings: List<StatisticDataDto>
)